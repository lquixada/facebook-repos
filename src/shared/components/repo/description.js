import React from 'react';
import styled from 'styled-components';

export const Description = ({text}) => (
  <P>{text}</P>
);

const P = styled.p`
  color: #393939;
  font-size: 1.4rem;
  line-height: 1.5;
`;
