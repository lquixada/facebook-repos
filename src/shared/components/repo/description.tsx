import React from "react";

import {P} from "../grid";

interface IProps {
  text: string | undefined;
}

export const Description = ({text}: IProps) => (
  <P>{text}</P>
);
