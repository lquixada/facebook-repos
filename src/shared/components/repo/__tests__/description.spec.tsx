import {mount} from 'enzyme'
import React from 'react'

import {Description} from '../description'

describe('<Description />', () => {
  it('matches the snapshot', () => {
    const component = mount(<Description text='some text' />)
    expect(component).toMatchSnapshot()
  })
})
