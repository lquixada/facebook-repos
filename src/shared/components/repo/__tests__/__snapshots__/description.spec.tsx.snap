// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Description /> matches the snapshot 1`] = `
<Description
  text="some text"
>
  <styled.p>
    <p
      className="sc-htoDjs BulxG"
    >
      some text
    </p>
  </styled.p>
</Description>
`;
